src/foam/nanos/export/ExportDriverRegistry.js

```diff
@@ -26,6 +26,18 @@ foam.CLASS({
       class: 'String',
       name: 'mimeType',
       value: 'text/csv;charset=utf-8'
+    },
+    {
+      class: 'Boolean',
+      name: 'canBeConverted'
```

Comment:
I would like to go with isConvertible, isDownloadable, and isOpenable. 
Unless it causes complications in the ExportModel below.

Reply:
Fixed.
Thank you